mmpose/datasets/pipelines/pose3d_transform.py

```diff
@@ -206,23 +312,50 @@ class RelativeJointRandomFlip:
     """Data augmentation with random horizontal joint flip around a root joint.
 
     Args:
-        item (str): The name of the pose to flip.
-        root_index (int): Root joint index in the pose.
-        visible_item (str): The name of the visibility item which will be
-            flipped accordingly along with the pose.
+        item (str|list[str]): The name of the pose to flip.
+        flip_cfg (dict|list[dict]): Configurations of the fliplr_regression
```

Comment:
I am afraid this is not clear enough. Here we should either clarify the meaning and data type for each item in flip_cfg, or refer to the docstring of the function "fliplr_regression".